src/mongocxx/test/collection.cpp

```diff
@@ -2406,13 +2406,15 @@ TEST_CASE("create_index tests", "[collection]") {
 
         options::index options{};
         options.unique(true);
+        options.hidden(true);
         options.expire_after(std::chrono::seconds(500));
         options.name(index_name);
 
         REQUIRE_NOTHROW(coll.create_index(keys.view(), options));
 
         bool unique = options.unique().value();
-        auto validate = [unique](bsoncxx::document::view index) {
+        bool hidden = options.hidden().value();
+        auto validate = [=](bsoncxx::document::view index) {
```

Comment:
So, the `[=]` copy-capture will capture all current variables by copy.  `[&]`, reference-capture, is a better choice here, because there aren't any lifetime issues with these variables that would require making a copy for the lambda.  The fewer copies we make the better!

Reply:
Done